commit 84c6853f3667b9b69a86177ed415c5b796cb704b
Author: Tamizh chelvam <c_traja@qti.qualcomm.com>
Date:   Thu Feb 2 08:32:18 2017 +0200

    ath10k: fix boot failure in UTF mode/testmode
    
    commit cb4281528b62207918b1e95827cad7527aa4dbaa upstream.
    
    Rx filter reset and the dynamic tx switch mode (EXT_RESOURCE_CFG)
    configuration are causing the following errors when UTF firmware
    is loaded to the target.
    
    Error message 1:
    [ 598.015629] ath10k_pci 0001:01:00.0: failed to ping firmware: -110
    [ 598.020828] ath10k_pci 0001:01:00.0: failed to reset rx filter: -110
    [ 598.141556] ath10k_pci 0001:01:00.0: failed to start core (testmode): -110
    
    Error message 2:
    [ 668.615839] ath10k_ahb a000000.wifi: failed to send ext resource cfg command : -95
    [ 668.618902] ath10k_ahb a000000.wifi: failed to start core (testmode): -95
    
    Avoiding these configurations while bringing the target in
    testmode is solving the problem.
    
    Signed-off-by: Tamizh chelvam <c_traja@qti.qualcomm.com>
    Signed-off-by: Kalle Valo <kvalo@qca.qualcomm.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1286_84c6853f3667b9b69a86177ed415c5b796cb704b.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1286_84c6853f3667b9b69a86177ed415c5b796cb704b.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
